src/tutorial/01-Setup/c.md

```diff
@@ -26,6 +26,20 @@ The following packages are also officially supported:
 > [sbt-launcher-package](https://github.com/sbt/sbt-launcher-package)
 > project.
 
+### Ubuntu and other Debian-based Linux Distributions
+
+Linux distributions based on Debian, such as Ubuntu use the DEB format, but usually you don't install your software from a local DEB file. Instead they come with package managers both for the command line (e.g. apt-get, aptitude) or with a graphical user interface (e.g. Synaptic).
+Luckily, they all use the same Advanced Packaging Tool (APT) conventions:
+They will check a number of configured repositories for packages to offer for installation. sbt binaries are published to Bintray, and conveniently Bintray provides an APT repository. You just have to add the repository to the places your package manager will check. So in the directory `/etc/apt/sources.list.d`, create a file `sbt.list` with the contents `deb http://dl.bintray.com/sbt/debian /`
+
+For example like this:
+
+    echo "deb http://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
```

Comment:
Does this require `apt-get update` afterwards? Also I think we should show `apt-get install sbt` above.